src/main/java/org/olf/folio/order/OrderImport.java

```diff
@@ -546,14 +546,13 @@ public JSONArray validateRequiredValues(MarcReader reader,String token, String b
 			    	logger.error(record.toString());
 			    	responseMessages.put(fundValidationResult);
 			    }
-			    return responseMessages;
-			    
 			}  catch(Exception e) {
 		    	logger.fatal(e.getMessage());
 		    	JSONObject responseMessage = new JSONObject();
 		    	responseMessage.put("error", e.getMessage());
 		    	responseMessage.put("PONumber", "~error~");
 		    	responseMessages.put(responseMessage);
+                        return responseMessages();
```

Comment:
Two questions:

1. Should this be:
   ```suggestion
		    	return responseMessages;
   ```
1. Don't we also want to drop the other early return on line 531?
https://github.com/cul-it/order-import-poc/blob/3e5dd3a5bda4fc22ed2772f1f3668b9a354d1949/src/main/java/org/olf/folio/order/OrderImport.java#L531

Reply:
Thanks @fereira. Are you sure about line 531?

It looks to me like it's returning the outer `responseMessages` JSONArray and not the inner `responseMessage` JSONObject, but would be good for you to double check because I have gotten tripped up by these names multiple times as I reviewed this.